learn_evm/evm_opcodes.md

```diff
@@ -178,7 +178,7 @@ The gas information is a work in progress. If an asterisk is in the Gas column,
 | `0xf6` - `0xf9`           | Unused         | -                                                                                                                                                            | -                                                                                                         |
 | [`0xfa`](#staticcall)     | STATICCALL     | Similar to CALL, but does not modify state                                                                                                                   | -                                                                                                         | 40           |
 | `0xfb`                    | Unused         | -                                                                                                                                                            | -                                                                                                         |
-| [`0xfd`](#revert)         | REVERT         | Stop execution and revert state changes, without consuming all provided gas and providing a reason                                                           | -                                                                                                         | 0            |
+| [`0xfd`](#revert)         | REVERT         | Stop execution and revert state changes, without consuming all provided gas and providing a reason                                                           | [EIP 140](https://github.com/axic/EIPs/blob/75121e281a8737f979a969394b4360d4d277c115/EIPS/eip-140.md)     | 0            |
```

Comment:
```suggestion
| [`0xfd`](#revert)         | REVERT         | Stop execution and revert state changes, without consuming all provided gas and providing a reason                                                           | [EIP 140](https://eips.ethereum.org/EIPS/eip-140)     | 0            |
```

Can we link to an "official" ethereum URL, to keep consistency with the rest of the table? (I'm providing one as an example, but the table linting will probably be off if you apply it as-is)